Driver.java

```diff
@@ -1,20 +1,70 @@
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
+
 public class Driver {
-        private static ArrayList<Room> rooms;
-        private static Player player;
-        // put something here
-        private static final Room startRoom = null;
+    private static ArrayList<Room> rooms;
+    private static Player player;
+    // put something here
+    private static Room startRoom = null;
+    private static Scanner input;
 
     public static void main(String[] args) {
-        Scanner in = new Scanner(System.in);
-        String userName = in.next();
+        input = new Scanner(System.in);
+        System.out.println("Enter your Name!");
+        String userName = input.nextLine();
+
+        makeStory();
+
         player = new Player(userName, startRoom, null);
-        
+
+        run();
+    }
+
+    public static void run() {
+        player.getRoom().enterRoom();
+
+        while(true) {
+            Room currRoom = player.getRoom();
+            if(player.timeIsUp()) {
+                System.out.println("Game Over!");
+                break;
+            }
+
+            currRoom.printOptions();
+            int choice = input.nextInt();//need error handling
```

Comment:
Could do input.next(), and have chooseOption(String, player) just use strings for options.